docs/evm/Build/wallets-and-configurations.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Wallets & Configurations
+sidebar_label: Wallets & Configurations
+sidebar_position: 4
+---
+
+# Wallets & Configurations
+
+This document shows how to integrate the FlowEVM Network programmatically with your Dapp via MetaMask.
+
+## Metamask
+
+Integrating additional networks into MetaMask can pose challenges for users who lack technical expertise and may lead to errors. Simplifying this process can greatly enhance user onboarding for your application. This guide demonstrates how to create a straightforward button within your frontend application to streamline the addition of the Berachain network to MetaMask.
+
+### EIP-3035 & MetaMask
+
+[EIP-3035](https://eips.ethereum.org/EIPS/eip-3085) is an Ethereum Improvement Proposal that defines an RPC method for adding Ethereum-compatible chains to wallet applications. Since March 2021 MetaMask has implemented that EIP as part of their MetaMask [Custom Networks API](https://consensys.io/blog/connect-users-to-layer-2-networks-with-the-metamask-custom-networks-api).
+
+### Network configuration
+
+To add the FlowEVM network to Metamask, we need to add the following network confgiuration:
+
+#### Previewnet Network
+
+```js
+export const FLOWEVM_PREVIEWNET_PARAMS = {
+    chainId: '0x2eb',
```

Comment:
Chain ID hex will need to change: https://github.com/onflow/flow-go/pull/5421

Tom has it here: https://discord.com/channels/613813861610684416/1210325991780651039/1210663907006480476

Reply:
Ah ok got it.  I pulled this from another page on the docs so I'll make a PR to fix that